pkgs/by-name/ok/okms-cli/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  okms-cli,
+  testers,
+}:
+
+buildGoModule rec {
+  pname = "okms-cli";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "ovh";
+    repo = "okms-cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-We1Aq9dOzEa7M6tG8kiVGfWhXfkpdZaJsJ5MCM/HZL4=";
+  };
+
+  vendorHash = "sha256-DyIqsvqTi6Yq8MADty/iSiDLgdd2vP/IDyCOm0yaQzk=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+    "-X main.commit=${src.rev}"
```

Comment:
Note: this will not be the commit hash. It is just the same rev value as set above.